### PR TITLE
[ssip]: Add HLD

### PR DESCRIPTION
**Pull request by nazariig**
Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>

### CODE REVIEW

File: doc/syslog/syslog-design.md [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+### 2.4.1 Config DB
+
+#### 2.4.1.1 SSIP table
+```abnf
+; defines schema for syslog table configuration attributes
+key = SYSLOG_SERVER|server_ip_address ; server IP address. Must be unique
+
+; field = value
+source  = ip-addr    ; source IP address
+port    = 1*5DIGIT   ; server port (0..65535)
```
Comment (eddyk-nvidia) [inline doc/syslog/syslog-design.md]: If only UDP is supported please specify "UDP port"

File: doc/syslog/syslog-design.md [hunk 2]
```diff
@@ -0,0 +1,559 @@
...
+ipv6-addr   = 6( h16 ":" ) ls32
+              /                       "::" 5( h16 ":" ) ls32
+              / [               h16 ] "::" 4( h16 ":" ) ls32
+              / [ *1( h16 ":" ) h16 ] "::" 3( h16 ":" ) ls32
+              / [ *2( h16 ":" ) h16 ] "::" 2( h16 ":" ) ls32
+              / [ *3( h16 ":" ) h16 ] "::"    h16 ":"   ls32
+              / [ *4( h16 ":" ) h16 ] "::"              ls32
+              / [ *5( h16 ":" ) h16 ] "::"              h16
+              / [ *6( h16 ":" ) h16 ] "::"
+ip-addr     = ipv4-addr / ipv6-addr
```
Comment (eddyk-nvidia) [inline doc/syslog/syslog-design.md:333]: ip-addr-family ?

File: doc/syslog/syslog-design.md [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+1. Renders `rsyslog.conf.j2` template with `sonic-cfggen` to generate a new `rsyslogd` config file
+2. Restarts `rsyslog` service which triggers `rsyslogd` to load a new config file
+
+### 2.3.2 SSIP parameters
+
+**SSIP will have the next parameter mapping:**
+
+###### Table 3: SSIP parameters
+
+| SONiC  | Rsyslogd | Schema                     |
```
Comment (nazariig) [inline doc/syslog/syslog-design.md]: Config DB Schema

Comment (nazariig) [inline doc/syslog/syslog-design.md:204]: Config DB Schema

File: doc/syslog/syslog-design.md (hunk 1 above)
Comment (nazariig) [reply doc/syslog/syslog-design.md]: @eddyk-nvidia done

File: doc/syslog/syslog-design.md (hunk 3 above)
Comment (nazariig) [reply doc/syslog/syslog-design.md]: @volodymyrsamotiy done

**Pull request closed without merging**